contracts/Voter.sol

```diff
@@ -239,7 +261,9 @@ contract Voter is IVoter {
         isAlive[_gauge] = true;
         _updateFor(_gauge);
         pools.push(_pool);
-        emit GaugeCreated(_gauge, msg.sender, _internal_bribe, _external_bribe, _pool);
+        Pair(_pool).setHasGauge(true); // may need to switch to IPair?
+        Pair(_pool).setExternalBribe(_wxbribe); // Changed this to wrapped external bribe  from
+        emit GaugeCreated(_gauge, msg.sender, _internal_bribe, _external_bribe, _wxbribe, _pool);
```

Comment:
https://github.com/Velocimeter/instruments/pull/5/files#r1110965391

Please amend according to the above comment